Glossary.md

```diff
@@ -0,0 +1,149 @@
+ Glossary of Technical Terms
+---
+
+## A
+- [Account Aliases](#account-aliases)
+- [Auto Associations](#asynchronous)
+- [Auto-Renew Accounts](#auto-renew-accounts)
+- [Address Book](#address-book)
+- [Application Accounts](#application-accounts)
+
+## B
+- [BLS](#bls)
+- [Block Proof](#block-proof)
+- [Block Signature](#block-signature)
+- [Block Stream](#block-stream)
+
+## C
+- [Civilian Accounts](#civilian-accounts)
+- [Config.txt](#config.txt)
+- [Congestion Pricing](#congestion-pricing)
+- [Consensus Time](#consensus-time)
+
+## D
+- [Delegate Call](#delegate-call)
+
+## E
+- [EOA](#eoa)
+
+## H
+- [Hollow Accounts](#hollow-accounts)
+
+## L
+- [Leaky Tests](#leaky-tests)
+
+## P
+- [Precompiles](#precompiles)
+
+## S
+- [Security Model V2](#security-model-v2)
+- [Sidecars](#sidecars)
+- [Synthetic Transactions](#synthetic-transactions)
+- [System Contracts](#system-contracts)
+- [System Transaction](#system-transaction)
+
+## T
+- [Token Allowance](#token-allowance)
+
+## W
+- [Weibar](#weibar)
+
+---
+
+## Address Book
+**Address Book**: The data structure that contains data for each consensus node in the network, including node ID, alias, IP address, and weight.
+
+## Account Aliases
+**Account Aliases**: A user-friendly identifier that can be classified as either a public key or an Ethereum Virtual Machine (EVM) address. Instead of inputting a long string of characters, users can use simpler and more memorable aliases.
+
+## Application Accounts
+**Application Accounts:** Used by developers and users interacting with applications deployed on Hedera. These accounts enable users to interact with the functionality provided by dApps, such as gaming, finance, or other decentralized services.
+
+## Auto Associations
+**Auto Associations**: The ability of accounts to automatically associate with tokens. An auto association slot is one or more slots you approve that allow tokens to be sent to your contract without explicit authorization for each token type. If this property is not set, you must approve each token before it is transferred to the contract for the transfer to be successful via the `TokenAssociateTransaction` in the SDKs.
+
+## Auto-Renew Accounts
+**Auto-Renew Accounts**: A feature designed to automatically renew the lifecycle of certain entities (like accounts, files, smart contracts, topics, or tokens) by funding their renewal fees. This feature is particularly useful for entities that require continuous operation over long periods, as it automates the process of keeping these entities active by periodically charging a linked auto-renew account.
+
+---
+
+## BLS
+**Boneh-Lynn-Shacham (BLS)**: A paired-elliptic-curve based signature algorithm with attractive characteristics for aggregate signatures (signatures produced by aggregating many individual signatures, and verifiable with a single public key).  BLS comes from the names of the authors of the initial research paper (Dan Boneh, Ben Lynn, Hovav Shacham).
```

Comment:
As long as you name the authors (great!) link the paper at [the author's site](https://hovav.net/ucsd/papers/bls04.html)?

Reply:
There are _two_ BLS, one for the signature algorithm, and another for the paired curve discovered to use with that algorithm.  Interestingly the authors are _not_ the same (except for Ben Lynn).

Here we should specify `BLS Algorithm` to differentiate from the `BLS Curve Family`.